mqtt/mqtt-edgehub/src/command.rs

```diff
@@ -97,7 +90,7 @@ impl CommandHandler {
         Ok(())
     }
 
-    async fn subscribe(&mut self, topics: &[String]) -> Result<bool, CommandHandlerError> {
+    async fn subscribe(&mut self, topics: &[String]) -> Result<(), CommandHandlerError> {
         let subscriptions = topics.iter().map(|topic| proto::SubscribeTo {
             topic_filter: topic.to_string(),
             qos: proto::QoS::AtLeastOnce,
```

Comment:
```rust
line 105
- let mut subacks: HashSet<String> = HashSet::from_iter(topics.to_vec());
+ let mut subacks = topics.to_vec();
```
Probably we don't even need a HashSet due to a small number of items in the list (<100)